SeriesCruncher/app/src/main/java/com/balazs_csernai/seriescruncher/seriesdetails/presenter/SeriesDetailsPresenterImpl.java

```diff
@@ -47,9 +57,9 @@ public void onStop() {
 
         @Override
         public void onSuccess(SeriesDetailsModel model) {
-            screen.setTitle(model.getTitle());
-            screen.setEpisodes((EpisodeListModel) converter.convert(model));
+            detailsModel = model;
             seriesLoader.loadPoster(model.getImageUrl(), posterCallbacks);
+            screen.setTitle(model.getTitle());
```

Comment:
The setter is soooo lonely here, let's make it happy by grouping it with the others.